src/utils/url.js

```diff
@@ -154,3 +154,131 @@ export function stringifyQuery(query) {
 
     return search
 }
+
+/* Import URL utils start */
+
+const ADMIN_URL_REGEXP = new RegExp(
+    '^(ftp|http|https)://(.*(.bgdi.ch|.geo.admin.ch|.swisstopo.cloud)|localhost:[0-9]{1,5})'
+)
+
+// todo use correct URLs
+const PROXY_URL = 'https://service-proxy.bgdi-dev.swisstopo.cloud'
```

Comment:
No need anymore for the proxy stuff, the provided URL must implements HTTPS and CORS.

Reply:
removed